specification/messaging/messaging-metrics.md

```diff
@@ -0,0 +1,257 @@
+# Semantic Conventions for Messaging Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Metric: `messaging.publish.duration`](#metric-messagingpublishduration)
+- [Metric: `messaging.receive.duration`](#metric-messagingreceiveduration)
+- [Metric: `messaging.deliver.duration`](#metric-messagingdeliverduration)
+- [Metric: `messaging.settle.duration`](#metric-messagingsettleduration)
+- [Metric: `messaging.published_messages.count`](#metric-messagingpublished_messagescount)
+- [Metric: `messaging.received_messages.count`](#metric-messagingreceived_messagescount)
+- [Metric: `messaging.settled_messages.count`](#metric-messagingsettled_messagescount)
+
+<!-- tocstop -->
+
+## Metric: `messaging.publish.duration`
+
+This metric is required.
```

Comment:
Is there a reason why this is the only metric that is required? None of the consumer metrics are required.

Reply:
good catch! all of them are required now (with some conditions)